### DISCUSSION TITLE
小米安装包管理组件安装应用时会误点

### DISCUSSION THREAD

**Discussion by Emperormummy** [aoguai/subscription#44] [General]
今天我发现安装应用的时候会自动点击取消更新，导致无法正常安装应用，禁用此应用规则后正常。望尽快修复！

**Comment by aoguai** [comment]
请提供快照

**Comment by aoguai** [comment]
- aoguai/subscription 4cde30b6120c0ab11e71093dccebf521fcac8c2d
  - 已排除安装包管理组件

如果还存在请提供快照，同时您应该到 issues 提交错误报告，而不是在讨论。
